packages/chaire-lib-frontend/src/services/objects/SelectedObjectsManager.ts

```diff
@@ -1,84 +1,174 @@
 /*
- * Copyright 2022, Polytechnique Montreal and contributors
+ * Copyright 2024, Polytechnique Montreal and contributors
  *
  * This file is licensed under the MIT License.
  * License text available at https://opensource.org/licenses/MIT
  */
 import events from 'events';
 
+type Idable = {
+    id: string;
+};
+
 // This manager keeps track of selected objects for the interface.
 // The selected object names must be known in advance (at construct)
 // TODO: The selected object manager is more the application state than a manager. It could be part of the application state or context instead
+
+/* Whenever a selection is added, removed or updated, all unrelated
```

Comment:
This is not really a comment on the class, should there be a FIXME or TODO?

Reply:
removed